packages/qdrant-loader/src/qdrant_loader/connectors/jira/cloud_connector.py

```
@@ -0,0 +1,138 @@
+            for issue in issues:
+                try:
+                    parsed_issue = self._parse_issue(issue)
+                    yield parsed_issue
+                    processed_count += 1
+
+                    if (processed_count) % progress_log_interval == 0:
+                        logger.info(
+                            f"🎫 Processed {processed_count} JIRA issues so far"
+                        )
+
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)